models/recharge.yml

```diff
@@ -45,8 +45,6 @@ models:
         description: The date and time the order was shipped.
       - name: address_id
         description: "{{ doc('address_id') }}"
-      - name: is_deleted
-        description: This column has been deprecated as of July 2025 in favor of the `_fivetran_deleted` column. Boolean indicating if the order (record) is deleted.
```

Comment:
This field removal needs to also be documented in the CHANGELOG to highlight the field was removed from the end model as well.

Reply:
Thank you updated.